test/errorTest.sh

```diff
@@ -2,34 +2,42 @@
 
 # Before any test, this command is executed
 oneTimeSetUp() {
-    ITERATIONS=1 BENCHES_SCRIPT_DIR=test/testError ./runAll.sh aDate
+    BENCHES_SCRIPT_DIR=test/testError ./runAll.sh aDate
```

Comment:
I'd put the ITERATIONS=4 here, and not in the benchmark.
Otherwise, the number of iterations is very far from the tests.

Also I'd not replace this test, I'd keep both, as said in the other comment.

Reply:
The thing is, if I make 2 benchmark files, only one would require the ITERATIONS=4, since the other one instead calls runBenchs 4 times. So if I put it in errorTest, then the other benchmark will produce 16 lines in the csv file.
I can adapt the unit tests though.